src/css/nav.css

```diff
@@ -68,10 +68,10 @@
   font-size: 0.9em;
   line-height: 2;
   padding: 0 1.5em 0 0.5em;
-  border: 1px solid #c1c1c1;
+  border: 1px solid #e5e5e5;
   border-radius: 0.25em;
   box-shadow: 0 1px 1px 1px rgba(0, 0, 0, 0.05);
-  background: url(data:image/svg+xml;charset=US-ASCII,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20448%20512%22%3E%3Cpath%20fill%3D%22%23333%22%20d%3D%22M207.029%20381.476L12.686%20187.132c-9.373-9.373-9.373-24.569%200-33.941l22.667-22.667c9.357-9.357%2024.522-9.375%2033.901-.04L224%20284.505l154.745-154.021c9.379-9.335%2024.544-9.317%2033.901.04l22.667%2022.667c9.373%209.373%209.373%2024.569%200%2033.941L240.971%20381.476c-9.373%209.372-24.569%209.372-33.942%200z%22%2F%3E%3C%2Fsvg%3E) no-repeat right 0.5em top 55% / 0.75em auto, linear-gradient(to bottom, #fff 0%, #e5e5e5 100%) repeat;
+  background: url(data:image/svg+xml;charset=US-ASCII,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%20448%20512%22%3E%3Cpath%20fill%3D%22%23333%22%20d%3D%22M207.029%20381.476L12.686%20187.132c-9.373-9.373-9.373-24.569%200-33.941l22.667-22.667c9.357-9.357%2024.522-9.375%2033.901-.04L224%20284.505l154.745-154.021c9.379-9.335%2024.544-9.317%2033.901.04l22.667%2022.667c9.373%209.373%209.373%2024.569%200%2033.941L240.971%20381.476c-9.373%209.372-24.569%209.372-33.942%200z%22%2F%3E%3C%2Fsvg%3E) no-repeat right 0.5em top 55% / 0.75em auto, #fff repeat;
```

Comment:
If we remove the linear gradient from this select element, we should remove it from the one in the search results as well. In other words, they should match.